libetrv/cli.py

```diff
@@ -70,6 +70,9 @@ def name(self):
     def current_time(self):
         time_utc = self._device.current_time
         print("Current time: {}".format(time_to_str(time)))
+        
+    def set_setpoint(self,setpoint)
```

Comment:
Please add missing space between function arguments to meet PEP8

Reply:
ok i do it